### DISCUSSION TITLE
First Developer Call

### DISCUSSION THREAD

**Discussion by vandan** [MetaMask/metamask-improvement-proposals#16] [Announcements]
Hey there, MetaMask developers!

Join us next Thursday (July 6th) to learn about a new MetaMask Improvement Proposal process we’re rolling out in order to provide transparency on planned API changes and to invite developers to make proposals as well. 🦊

We’ll be covering:

- Overview and contribution guidelines for the MetaMask Improvement Proposals 
- Review of in-progress proposals
- Call for comments on [MIP-1](https://github.com/MetaMask/metamask-improvement-proposals/blob/main/MIPs/mip-1.md), which extends wallet_watchAsset to support NFTs
- We look forward to seeing you in our very first MetaMask API developer call! 

JOIN [HERE](https://www.google.com/url?q=https://community.consensys.io/events/details/consensys-global-events-presents-metamask-api-developer-call/&sa=D&source=calendar&ust=1688422177957394&usg=AOvVaw1LRVwRnpTrB6XMOEz68Nda).

Cheers! 🎉

**Comment by shanejonas** [comment]
if there is time, we can chat a bit about the versioning MIP: https://github.com/MetaMask/metamask-improvement-proposals/pull/11

**Comment by vandan** [comment]
In case anyone couldn't join live, here's [the replay of the introduction](https://www.youtube.com/watch?v=YxIgYWK9294) to the [MetaMask Improvement Proposals](https://github.com/MetaMask/metamask-improvement-proposals).
